src/groovy/org/pih/warehouse/core/RoleType.groovy

```diff
@@ -22,6 +22,7 @@ enum RoleType {
     // Complementary roles that provide additional functionality
     ROLE_FINANCE('Financial User', 100),
     ROLE_INVOICE('Invoice user', 100),
+    ROLE_PRODUCT_MERGE('Product Merge', 100),
```

Comment:
I don't remember us discussing about the `sortOrder` for this role so I assumed it to be same **(100)** as other _"complementary roles that provide additional functionality"_

Reply:
Yes this seems fine for now. Only the primary roles are sorted at the moment so this isn't a big deal.